features/fields_extensie.feature

```diff
@@ -1,126 +1,570 @@
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                           |
+      | fields   | fields        | Parameter bevat een niet bestaande propertynaam. |
+
+    Scenario: Fields parameter met attribuutnaam met onjuiste case
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                   |
+      | type                | RaadpleegMetBurgerservicenummer          |
+      | burgerservicenummer | 999994086                                |
+      | fields              | BurgerServiceNummer,geslachtsaanduiding  |
+      Dan heeft de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                                |
+      | fields   | fields        | Parameter bevat een porpertynaam met onjuiste casing. |
+
+  Rule: properties die met dezelfde naam in meerdere groepen voorkomen moeten met een .(dot)-notatie worden opgenomen in fields
```

Comment:
misschien iets meer uitleggen in de regel wat de dot notatie is? Of moet ik dat alleen uit de voorbeelden kunnen halen?
je zegt nu wanneer dit moet. niet dat het altijd mag

Reply:
het gaat om een "moeten". dan zou ik eigenlijk een fout-scenario verwachten die de foutmelding toont die je krijgt wanneer de propertynaam in fields niet uniek is